community-membership.md

```diff
@@ -1,77 +1,230 @@
 # Community membership
 
-New community members:
-
-- [**New Contributor**](https://github.com/kubernetes/contrib/issues/1090): a
-  couple of PRs; should be welcomed to the community, helped with PR workflow, and
-  directed to relevant documentation
-- **Active Contributor**: at least 3 merged and/or assigned PRs (which could include documentation
-  contributions as well as code), including one in the past month; we have
-  [expectations](contributors/devel/community-expectations.md)
-  that frequent contributors will assist in our code-review process and with project
-  maintenance
-  
-Established community members:
+This doc outlines the various responsibilities of contributor roles on the OSS Kubernetes.  The Kubernetes
+project is subdivided into various subareas.  Responsibilities for many contributor roles are
+scoped by these subareas.
+
+Summary of the most common roles on the project:
+
+| Role | Responsibilities | Requirements | Defined by |
+| -----| ---------------- | ------------ | -------|
+| Member | Active contributor in the community | Sponsored by 2 members.  Already active | Member of kubernetes GitHub org |
+| Reviewer | Review contribution from other members | History of review and authorship in an area | Reviewer entry in an OWNER file |
+| Approver | Approve accepting contributions | Highly experienced and active reviewer + contributor to an area | Approver entry in an OWNER file|
+| Owner | Set priorities and approve proposals | Primary or major contributor and maintainer of area | Approve entry in a top-level area OWNER file |
```

Comment:
Yes, how the role is recorded in OWNERS files is not a hard definition, and is loosely defined by being an approve in the top-level OWNERS file for the area.

Yes, the difference in role  responsibility is that the OWNER "Act as one of the decision makers for the area", and are the first escalation point for decisions within the area.  

Will try to improve the language here.